hbase-client/src/main/java/org/apache/hadoop/hbase/NotAllRootRegionsOnlineException.java

```diff
@@ -25,19 +25,19 @@
  * Thrown when an operation requires the root and all meta regions to be online
  */
 @InterfaceAudience.Public
-public class NotAllMetaRegionsOnlineException extends DoNotRetryIOException {
+public class NotAllRootRegionsOnlineException extends DoNotRetryIOException {
```

Comment:
Should this be named ROOTNotOnlineException  or RootRegionNotOnlineException(singular)

Reply:
Yes good point.